druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/query/lookup/namespace/RocksDBExtractionNamespace.java

```diff
@@ -13,7 +13,7 @@
 import javax.validation.constraints.NotNull;
 import java.util.Objects;
 
-@JsonTypeName("maharocksdb")
+@JsonTypeName("mahainmemorydb")
```

Comment:
This is not a backwards compatible change.  What is the rationale for this change?